docs/logger-setup-in-zio-application.md

```diff
@@ -0,0 +1,36 @@
+---
+id: logger-setup-in-zio-application
+title: "Logger setup in ZIO application"
+---
+
+The recommended place for setting the logger is application boostrap.
+In this case, custom logger will be set for whole application runtime (also application failures will be logged with specified logger).
+
+[//]: # (TODO: make snippet type-checked using mdoc)
+
+```scala
+import zio.logging.{ LogFormat, console }
+import zio.{ ExitCode, Runtime, Scope, ZIO, ZIOAppArgs, ZIOAppDefault, ZLayer }
+
+object SimpleApp extends ZIOAppDefault {
+
+  override val bootstrap: ZLayer[ZIOAppArgs, Any, Any] =
+    Runtime.removeDefaultLoggers >>> console(LogFormat.default)
+
+  override def run: ZIO[Scope, Any, ExitCode] =
+    for {
+      _ <- ZIO.logInfo("Start")
+      _ <- ZIO.fail("FAILURE")
+      _ <- ZIO.logInfo("Done")
+    } yield ExitCode.success
+
+}
+```
+
+Expected console output:
+
+```
+timestamp=2022-10-28T18:40:25.517623+02:00 level=INFO thread=zio-fiber-6 message="Start"
+timestamp=2022-10-28T18:40:25.54676+02:00  level=ERROR thread=zio-fiber-0 message="" cause=Exception in thread "zio-fiber-6" java.lang.String: FAILURE
+	at zio.logging.example.SimpleApp.run(SimpleApp.scala:14)
+```
```

Comment:
this i think is good to move to index.md and not have it like separated page

Reply:
Currently, we have a (somehow) unique template for the index.md page:

```
# PROJECt_TITLE

DESCRIPTION

PROJECT_BADGES

## Introduction

## Installation

## Example

```

So one example suffices on the index page. Let me know which example is better for that.

Reply:
I removed the outdated example. Done!

Reply:
I removed the outdated example. Done!